src/Visualization/Display3d.cpp

```diff
@@ -46,28 +46,18 @@ void Display3d::Init(long window_handle,
   printf("Aspect_DisplayConnection created.\n");
   if (GetGraphicDriver().IsNull())
   {
-  GetGraphicDriver() = new OpenGl_GraphicDriver (aDisplayConnection);
+    GetGraphicDriver() = new OpenGl_GraphicDriver (Handle(Aspect_DisplayConnection)());
```

Comment:
@tpaviot, this is an old commit, these changes were due to poor rebasing from my side and have been reverted to your recent changes. sorry for the noise...